functions/Get-DbaSsisEnvironment.ps1

```diff
@@ -0,0 +1,229 @@
+
+Gets variables of environments other than 'DEV' and 'PROD' located in folders other than 'DWH_ETL', 'DEV2' and 'QA' on 'localhost' server
+
+.NOTES
+Author: Bartosz Ratajczyk ( @b_ratajczyk )
+
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+#>
+function Get-DbaSsisEnvironment {
+
+[CmdletBinding()]
+	Param (
+		[parameter(Mandatory, ValueFromPipeline)]
+		[Alias('SqlServer', 'ServerInstance')]
+		[DbaInstanceParameter[]]$SqlInstance,
+        [Parameter(Mandatory=$false)]
+        [System.Management.Automation.PSCredential]$SqlCredential,
+        [parameter(Mandatory=$false)]
+		[object[]]$Environment,
+        [parameter(Mandatory=$false)]
+        [object[]]$EnvironmentExclude,
+        [parameter(Mandatory=$false)]
+        [object[]]$Folder,
+        [parameter(Mandatory=$false)]
+        [object[]]$FolderExclude,
+		[switch]$Silent
+	)
+
+    begin {
+
+    }
+
+    process {
+
+        foreach($instance in $SqlInstance)
+        {
+            try
+            {
+                Write-Message -Message "Connecting to $instance" -Level 5 -Silent $Silent
+                $connection = Connect-SqlInstance -SqlInstance $instance
+            }
+            catch
+            {
+                Stop-Function -Message "Failed to connect to: $instance" -ErrorRecord $_ -Target $instance -Continue -Silent $Silent
+                return
+            }
+            
+
+            if ($connection.versionMajor -lt 11)
+            {
+                Stop-Function -Message "SSISDB catalog is only available on Sql Server 2012 and above, exiting." -Silent $Silent
+                return
+            }
+
+            try
+            {
+                $ISNamespace = "Microsoft.SqlServer.Management.IntegrationServices"
+
+                Write-Message -Message "Connecting to $instance Integration Services" -Level 5 -Silent $Silent
+                $SSIS = New-Object "$ISNamespace.IntegrationServices" $connection
+            }
+            catch
+            {
+                Stop-Function -Message "Could not connect to Integration Services on $instance" -Silent $Silent
+                return
+            }
+
+            Write-Message -Message "Fetching SSIS Catalog and its folders" -Level 5 -Silent $Silent
```

Comment:
My mistake. Should be 'SSIS Catalog', will change it and use 'SSIS Catalog' everywhere as it's the name one can find in docs